lib/generators/templates/sincerely.yml

```diff
@@ -7,6 +7,7 @@ defaults: &defaults
         region: <%%= ENV['AWS_REGION'] %>
         access_key_id: <%%= ENV['AWS_ACCESS_KEY_ID'] %>
         secret_access_key: <%%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+        configuration_set_name: config_set
```

Comment:
What's this configuration used for?

Reply:
it is needed for config the SES webhook, the configuration_set_name is sent when the ses email is being generated